src/python/espressomd/particle_data.pyx

```diff
@@ -1888,7 +1888,7 @@ cdef class ParticleList(object):
             for p in self:
                 for t in types:
                     if (p.type == t or t == "all"):
-                        vtk.write("{} {} {}\n".format(*(p.pos % box_l)))
+                        vtk.write("{} {} {}\n".format(*(p.pos_folded)))
```

Comment:
What's the difference?

Reply:
pos_folded includes the Lees Edwards shift, whereas pos % box_l doesn't